docs/design.md

```diff
@@ -967,6 +864,92 @@ spare capacity is chosen in the same datacenter and a special-case split
 is done which simply duplicates the data 1:1 and resets the range
 configuration metadata.
 
+# Raft - Consistency of Range Replicas
+
+Each range is configured to consist of three or more replicas, as specified by
+their ZoneConfig. The replicas in a range maintain their own instance of a
+distributed consensus algorithm. We use the [*Raft consensus algorithm*](https://ramcloud.stanford.edu/wiki/download/attachments/11370504/raft.pdf)
```

Comment:
https://raftconsensus.github.io/ is probably a better link (or if you want to link directly to the paper, it's available at the shorter url http://ramcloud.stanford.edu/raft.pdf).